pageserver/src/walingest.rs

```diff
@@ -2177,21 +2178,30 @@ mod tests {
         let pg_version = 15; // The test data was generated by pg15
         let path = "test_data/sk_wal_segment_from_pgbench";
         let wal_segment_path = format!("{path}/000000010000000000000001.zst");
+        let source_initdb_path = format!("{path}/{INITDB_PATH}");
         let startpoint = Lsn::from_hex("14AEC08").unwrap();
         let endpoint = Lsn::from_hex("1FFFF98").unwrap();
 
+        let harness = TenantHarness::create("test_ingest_real_wal").unwrap();
+        let (tenant, ctx) = harness.load().await;
+        let tenant_shard_id = tenant.tenant_shard_id();
+
+        // This instrumentation is needed to bootstrap the timeline
+        let span = tracing::info_span!(parent: None, "test_ingest_real_wal", tenant_id=%tenant_shard_id.tenant_id, shard_id=%tenant_shard_id.shard_slug(), timeline_id=%TIMELINE_ID);
+        let _enter = span.enter();
```

Comment:
I was initially going to comment about use of `Span::enter` in async context but then saw that the added code is blocking...

Why add the span? it cannot be any other test running on these lines?

Reply:
I now use `tracing::instrument`

Reply:
Looking good.